routes/api/contacts.js

```diff
@@ -1,25 +1,77 @@
-const express = require('express')
+const express = require("express");
+const crypto = require("crypto");
+const router = express.Router();
+const contacts = require("../../models/contacts");
+const { HttpError } = require("../../utils");
+const Joi = require("joi");
 
-const router = express.Router()
+const reqSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
Schema must be in separate file. You should validate email and phone properly

Reply:
fixed:
- reqSchema moved into utils folder as a standalone file;
- changed validation logic for phone and email fields by adding regex and tilds;